htlcswitch/test_utils.go

```diff
@@ -941,13 +941,14 @@ func createClusterChannels(aliceToBob, bobToCarol btcutil.Amount) (
 // newThreeHopNetwork function creates the following topology and returns the
 // control object to manage this cluster:
 //
-//	alice			   bob				   carol
-//	server - <-connection-> - server - - <-connection-> - - - server
-//	 |		   	  |				   |
-//   alice htlc			bob htlc		    carol htlc
-//     switch			switch	\		    switch
-//	|			 |       \			|
-//	|			 |        \			|
+//		alice			   bob				   carol
+//		server - <-connection-> - server - - <-connection-> - - - server
+//		 |		   	  |				   |
+//	  alice htlc			bob htlc		    carol htlc
+//	    switch			switch	\		    switch
+//		|			 |       \			|
+//		|			 |        \			|
+//
```

Comment:
I think this comment needs some manual intervention, as not all parts of the ASCII graphic were formatted. Same further down in this file.

Reply:
yeah the new formatter doesn't like ASCII art, it just treats them as code blocks. Fixed the alignment.